rfcs/protocols/asset-exchange/corda-htlc-alternatives.md

```diff
@@ -0,0 +1,68 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+ 
+# Design Choices for Asset Exchange - HTLC Protocol Units in Corda Networks
+
+- RFC: 02-006
+- Authors: Ermyas Abebe, Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
+- Status: Proposed
+- Since: 25-Nov-2020
```

Comment:
I think date refers to last updated date, so probably update that in all rfcs where you have made changes

Reply:
Done.